### ISSUE TITLE
Nudge Boat Exploit

### ISSUE THREAD

**Issue by AdventuresOfSigmundr** [blaxxun-boop/Sailing#6]
You can continuously nudge the boat from the rear right seat for a major speed boost while sailing.

**Comment by Ripstein00** [comment]
Yeah you can sit in the boats and push them using the ladder to boost them forward like a rocket.  You can make the boat go faster against the wind than it normally does with full sail.  I wonder if it would be possible to make it so that the boats couldn't be pushed while standing in them..
